docs/v3/examples/flags.md

```diff
@@ -106,6 +106,8 @@ See full list of flags at https://pkg.go.dev/github.com/urfave/cli/v3
 
 For bool flags you can specify the flag multiple times to get a count(e.g -v -v -v or -vvv)
 
+> If you want to support the `-vvv` flag, you need to set `Command.UseShortOptionHandling`.
```

Comment:
I would try to rephrase it. Currently it sounds like only _the_ `-vvv` flag will be supported if you set `UseShortOptionHandling` to true.

Reply:
Suggestion:

To enable the parsing of combined flags (e.g., `-vvv` as `-v -v -v` or `-xfc` as `-x -f -c` ...etc.), you need to set `Command.UseShortOptionHandling`.

Reply:
Is it too late to rename it to `Command.ShortOptionsJoined`

Reply:
I think not, it's still alpha, it's the right time to make such changes.

But I'd rename it to `JoinShortOption`. Even shorter:P